src/components/ProductListed/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Heart from 'components/shared/Icons/Heart';
+import Button, { ButtonSize, ButtonVariants } from 'components/shared/Buttons';
+
+export enum ProductTypes {
+  Restored = 'Restored',
+  New = 'New',
+}
```

Comment:
Shouldn't these match to the strings in the array I commented above? Shouldn't "Used" be here?